interface/globals.php

```diff
@@ -209,11 +209,22 @@ function undoMagicQuotes($array, $topLevel=true) {
         }
       }
     }
-    if ($gl_name == 'language_menu_other') {
+    if ($gl_name == 'language_menu_other') {       
       $GLOBALS['language_menu_show'][] = $gl_value;
     }
     else if ($gl_name == 'css_header') {
       $GLOBALS[$gl_name] = "$rootdir/themes/" . $gl_value;
+      
+      // For RTL languages, substitute the same theme converted for RTL// 
+      if ( isset( $_SESSION['language_direction'] ) &&
+              $_SESSION['language_direction'] == 'rtl' && 
+              !strpos($GLOBALS['css_header'], 'rtl')  ) {
+        $GLOBALS['css_header'] = "$rootdir/themes/rtl_" . $gl_value; // str_replace( 'themes/style_', 'themes/rtl_style_', $GLOBALS['css_header'] );
```

Comment:
https://github.com/matrix-olegsv/openemr/commit/3397d88bc7dae2b93db898375a60a3130fcf71ee

As you requested, I have added file exists check to globals.php . 


As for login.php I believe it deserves additional treatment due to a number of issues not directly related to language direction. 

- first, if the user is already logged in and the session is valid, the login screen should detect this and redirect the user to the home screen.

- second, today when user enters wrong credentials AND changes the language, then when the user gets error message then language is reset and the user sees the error in default language.

I would suggest to leave login screen as it is for now, and create a  separate fix to address aforementioned problems.